infection.json5.dist

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "vendor/infection/infection/resources/schema.json",
+    "testFrameworkOptions": "--testsuite=unit",
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "mutators": {
+        "@default": true,
+        "Throw_": {
+            "ignore": [
+                "PhpCsFixer\\Cache\\FileHandler"
+            ]
+        }
+    },
+    "logs": {
+        "text": "dev-tools/infection/run.log",
+        "html": "dev-tools/infection/report.html"
```

Comment:
I think you don't need it on CI, neither `text` nor `html`

Reply:
I wanted to create them for local runs, so they can be viewed and analysed. I will take a look how to do it better.

For now I need to figure out what's the problem in the CI, but I'll take a look later.

Reply:
Since it's impossible to add these reports dynamically with CLI options, I'll leave it here because for diff-based testing in CI it shouldn't be an too big overhead.

Reply:
In the end I removed them from dist config, everyone can add them to local config if needed 🙂.